src/com/cinergix/mapper/ObjectMapper.java

```diff
@@ -216,8 +216,16 @@ protected boolean checkColumnLabelExist( ResultSet result, String columnLabel ){
 		try{
 			
 			return result.findColumn( columnLabel ) > 0;
+			
 		}catch( SQLException ex ){
-			// This exception thrown only when there is no matching column found in the result set.
+			
+			if( ex.getMessage().matches( "Column(.*)not\\s+found\\." ) ){
+				// This error will throw when the column is not found in given result.
+				// TODO This exception is handled based on the message in the exception. This can be improved by
+				// finding better way to identify the cause of exception.
```

Comment:
Comment should Explain that the specific message is expected when filtering the columns. TODO is not necessary